kolibri/core/analytics/pskolibri/_pswindows.py

```diff
@@ -0,0 +1,316 @@
+"""Windows platform implementation."""
```

Comment:
This seems quite complicated :) How has it been tested? Is the code from elsewhere or did you write it from scratch?

Reply:
This is code from psutil